src/components/entity/list.vue

```diff
@@ -572,12 +574,17 @@ export default {
       this.odataEntities.value?.forEach(e => { e.__system.selected = false; });
       this.allSelected = false;
     },
+    cancelBackgroundRefresh() {
+      this.odataEntities.cancelRequest();
+      this.deletedEntityCount.cancelRequest();
+    },
```

Comment:
```suggestion
    cancelBackgroundRefresh() {
      if (!this.refreshing) return;
      this.odataEntities.cancelRequest();
      this.deletedEntityCount.cancelRequest();
    },
```

What do you think about adding an early return to `cancelBackgroundRefresh()`? I know in most cases, such a check isn't necessary, but I think it makes it a little clearer to reason about the method and its possible side effects. I started thinking, even if a background refresh isn't in progress, I think it's possible for the initial request for `this.deletedEntityCount` to be in progress when `cancelBackgroundRefresh()` is called, in which case the request for `this.deletedEntityCount` will be canceled prematurely/unnecessarily. I know that case is pretty unlikely, but I think we can circumvent situations like that by checking `this.refreshing` upfront.